specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/securitydomain.json

```diff
@@ -185,6 +185,23 @@
     }
   },
   "definitions": {
+    "MofNCertificateInfoObject": {
```

Comment:
If we ever did generate a model, we don't use abbreviations in class names. You could either rename this to something more human readable, or leave the model name, but add an `x-ms-client-name`: https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-client-name